src/db_iter.h

```diff
@@ -76,10 +98,11 @@ class TitanDBIterator : public Iterator {
   }
 
  private:
-  void GetBlobValue() {
+  // returns whether the value is in blob file or not
```

Comment:
This bool return isn't clear enough for me. IMO any meaning other than status is confusing. And the NUM_SEEK seems to refer to attempted seek on blob which requires more clarification. In this context I think a test on `iter->IsBlob()` is enough to RecordTick.